tests/foreman/cli/test_discoveredhost.py

```diff
@@ -79,36 +76,34 @@ def setUpClass(cls):
         )
 
         # Create Org and location
-        cls.org = make_org()
-        cls.loc = make_location()
+        org = make_org()
+        loc = make_location()
 
         # Get default settings values
-        cls.default_discovery_loc = Settings.list({'search': 'name=%s' % 'discovery_location'})[0]
-        cls.default_discovery_org = Settings.list(
-            {'search': 'name=%s' % 'discovery_organization'}
-        )[0]
-        cls.default_discovery_auto = Settings.list({'search': 'name=%s' % 'discovery_auto'})[0]
+        default_discovery_loc = Settings.list({'search': 'name=%s' % 'discovery_location'})[0]
+        default_discovery_org = Settings.list({'search': 'name=%s' % 'discovery_organization'})[0]
+        default_discovery_auto = Settings.list({'search': 'name=%s' % 'discovery_auto'})[0]
```

Comment:
use f-strings here

Reply:
f-strings? :thinking: this is just merging 2 strings. I could use one string instead, but I think someone tried to make it more  readable by separating it.

Reply:
It became part of our coding standards, and most of these uses cases have been converted. Its a great time to convert this one as well, I don't think it will reduce the readability since an f-string isn't even needed here.

`Settings.list({'search': 'name=discovery_location'})[0]`